README.md

```diff
@@ -200,3 +200,11 @@ Module Long Term Support (LTS)](http://github.com/CloudNativeJS/ModuleLTS) polic
 | 4.x     | End-of-Life     | Dec 2016  | Apr 2019 |
 
 Learn more about our LTS plan in [docs](https://loopback.io/doc/en/contrib/Long-term-support.html).
+
+## swagger-ui vulnerabilities
+
+There has been a security vulnerability on one of our dependencies `swagger-ui`: https://www.npmjs.com/advisories/985. 
```

Comment:
To me, the phrase "there has been" implies that those security vulnerabilities are gone. That's not the case, they are still present in the 2.x version of swagger-ui. Let's find a better phrasing please.

Also IIRC, there are more than a single known vulnerability, see the discussion in https://github.com/strongloop/loopback-component-explorer/issues/263. Let's make it more clear please which of the vulnerabilities are known by us and considered as not affecting LoopBack users.

Reply:
I've reworked the sentence a bit.